sysutils/cfiles/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        mananapr cfiles 1.5 v
+categories          sysutils
+platforms           darwin
+license             MIT
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+
+description         A ncurses file manager written in C with vim like keybindings
+long_description    ${name} is a terminal file manager with vim like keybindings, \
+                    written in C using the ncurses library. It aims to provide an \
+                    interface like ranger while being lightweight, fast and minimal.
+
+homepage            https://github.com/mananapr/cfiles
+
+checksums           rmd160  47bef44dbbedf89ced7ce2a87a9924f52695afbd \
+                    sha256  fb6c4a392d6e5dfa4d8bbc5dd4af332fcb4c26d91010d9b11f98662952fe6768 \
+                    size    237355
+
+patchfiles          patch-config.h.diff \
+                    patch-Makefile.diff
+
+use_configure       no
```

Comment:
When you do this, you need to make sure that you are [using the right compiler](https://trac.macports.org/wiki/UsingTheRightCompiler).

Reply:
Fixed.